templates/liar/templates.yaml

```diff
@@ -0,0 +1,78 @@
+dataset: liar
+templates:
+  331fec6f-bc64-4a81-b895-b1c5d5a61e7e: !Template
+    id: 331fec6f-bc64-4a81-b895-b1c5d5a61e7e
+    jinja: 'What was the party affiliation of the speaker of the following statement?
```

Comment:
It might be hard for the model to free-form guess a party affiliation - can you create a variant of this prompt where you list out the possible choices for the party affiliation? Or is this free-form, i.e. there is no finite set of party affiliations to choose from?

Reply:
Actually, there is no finite set of party affiliations to choose from. So it is fine to have it the way it is?

Reply:
Yep, that's fine.